### PR TITLE
C24_WMDE_Desktop_DE_04

### PR DESCRIPTION
**Pull request by gbirke**
Form parameters for different address data flow

Ticket: https://phabricator.wikimedia.org/T371962

### CODE REVIEW

File: campaign_info.toml [hunk 1]
```diff
@@ -6,24 +6,24 @@
 [desktop]
 name = "Desktop"
 icon = "desktop"
-campaign = "C24_WMDE_Desktop_DE_03"
-description = "Based on CTRL of C24_WMDE_Desktop_DE_02"
-campaign_tracking = "03-ba-240808"
+campaign = "C24_WMDE_Desktop_DE_04"
+description = "Test different donation forms"
```
Comment (Abban) [inline campaign_info.toml]: It's nice to also have what banner it's based on in the description as when we start jumping tests in the campaign it'll give us an in repo indicator of where a test came from.

**Review by Abban — APPROVED** [review]

File: banners/desktop/C24_WMDE_Desktop_DE_04/styles/Banner.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+@use 'src/themes/Treedip/variables/fonts';
```
Comment (Abban) [inline banners/desktop/C24_WMDE_Desktop_DE_04/styles/Banner.scss]: I just noticed that the imports in all these files point to the wrong theme. Can you fix them?

**Review by Abban — CHANGES_REQUESTED** [review]

File: campaign_info.toml [hunk 3]
```diff
@@ -6,24 +6,24 @@
 [desktop]
 name = "Desktop"
 icon = "desktop"
-campaign = "C24_WMDE_Desktop_DE_03"
-description = "Based on CTRL of C24_WMDE_Desktop_DE_02"
-campaign_tracking = "03-ba-240808"
+campaign = "C24_WMDE_Desktop_DE_04"
+description = "Based on C24_WMDE_Desktop_DE_04 ctrl, var: leads to different fundraising donation form (different address completion)"
```
Comment (Sperling-0) [inline campaign_info.toml]: Based on `C24_WMDE_Desktop_DE_04 ctrl`

**Review by Sperling-0 — CHANGES_REQUESTED** [review]

File: campaign_info.toml [hunk 4]
```diff
@@ -6,24 +6,24 @@
 [desktop]
 name = "Desktop"
 icon = "desktop"
-campaign = "C24_WMDE_Desktop_DE_03"
-description = "Based on CTRL of C24_WMDE_Desktop_DE_02"
-campaign_tracking = "03-ba-240808"
+campaign = "C24_WMDE_Desktop_DE_04"
+description = "Based on C24_WMDE_Desktop_DE_04_ctrl, var: leads to different fundraising donation form (different address completion)"
```
Comment (Sperling-0) [inline campaign_info.toml]: It is based on `C24_WMDE_Desktop_DE_03_ctrl`! (The typo is still not fixed for me 🙈)

Comment (gbirke) [reply campaign_info.toml]: So sorry, I thought you meant an underscore, did not see the wrong number 🤦‍♂️

**Review by Abban — APPROVED** [review]

**Pull request merged**